cob_base_velocity_smoother/mainpage.dox

```diff
@@ -2,9 +2,9 @@
 \mainpage
 \htmlinclude manifest.html
 
-\b cob_base_velocity_smoother is ... 
+\b cob_base_velocity_smoother is ...
```

Comment:
btw, what about those (leftover?) `mainpage.dox` files?
Should we remove those as well? Most of them seem to be empty anyways...